core/templates/pages/about-page/about-page.module.ts

```diff
@@ -65,11 +65,12 @@ class AboutPageModule {
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { downgradeModule } from '@angular/upgrade/static';
 
-const bootstrapFn = (extraProviders: StaticProvider[]) => {
+const bootstrapFnAsync = async(extraProviders: StaticProvider[]) => {
```

Comment:
do we need a space after async here?

Reply:
Adding a space after the 'async' keyword gave lint errors. (As mentioned by Srijan above, as it is an anonymous function we are unable to add space after it)

Reply:
Ah, I see! Thanks for the explanation! :)